website/style/index.css

```diff
@@ -235,3 +235,22 @@ a {
 #content p {
   color: var(--secondary-font-color);
 }
+
+table{
+  border-collapse: collapse;
+}
+th,
+td{
+  text-align: left;
+  padding: 12px;
+  border: 1px solid var(--border-color, #000);
+  border-collapse: collapse;
+  vertical-align: top;
+}
+th:nth-child(3),
+td:nth-child(3){
+  text-align: center;
+}
+td:nth-child(4){
+  white-space: nowrap;
+}
```

Comment:
Are these still needed?

Reply:
Yes, otherwise the table looks like this:
![image](https://user-images.githubusercontent.com/1891935/66068424-92b93600-e51b-11e9-97bd-c39debd6a0c7.png)